main.nf

```diff
@@ -484,29 +487,32 @@ process FEATURE_ENGINEERING_PART1 {
     """
 }
 
-process FEATURE_ENGINEERING_PART2 {
+process JOIN_TIER_PHRANK_PART1 {
+    tag "${scores.simpleName}"
+
     input:
-    path scores, stageAs: "scores.csv"
+    path scores
     path phrank
 
     path ref_annot_dir
     path ref_var_tier_dir
     path ref_merge_expand_dir
 
     output:
-    path "${scores.baseName}_scores.txt.gz", emit: compressed_scores
-    path "${scores.baseName}_Tier.v2.tsv", emit: tier
+    path "${scores.simpleName}_scores.txt.gz", emit: compressed_scores
+    path "${scores.simpleName}_Tier.v2.tsv", emit: tier
 
     script:
     """
+    mv $scores scores.csv
     VarTierDiseaseDBFalse.R ${params.ref_ver}
     generate_new_matrix_2.py ${params.run_id} ${params.ref_ver}
-    mv scores.txt.gz  ${scores.baseName}_scores.txt.gz
-    mv Tier.v2.tsv ${scores.baseName}_Tier.v2.tsv
+    mv scores.txt.gz  ${scores.simpleName}_scores.txt.gz
+    mv Tier.v2.tsv ${scores.simpleName}_Tier.v2.tsv
     """
 }
 
-process MERGE_RESULTS {
+process JOIN_TIER_PHRANK_PART2 {
```

Comment:
MERGE_SCORES would be better.

Reply:
I think we can still use "merge"

Reply:
I get it. Actually PART1 PART2 is not meaningful neither.